ibis/backends/tests/test_temporal.py

```diff
@@ -383,20 +383,53 @@ def test_temporal_binop_pandas_timedelta(
 @pytest.mark.parametrize(
     'comparison_fn',
     [
-        lambda t: t.timestamp_col > pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col >= pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col < pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col <= pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col == pd.Timestamp('20100301'),
-        lambda t: t.timestamp_col != pd.Timestamp('20100301'),
+        param(
+            lambda col: (
+                col > pd.Timestamp('20100301', tz="UTC").to_pydatetime()
```

Comment:
Maybe factor out the datetime value to a global constant?

Reply:
Done.